app/controllers/admin/welcome_controller.rb

```diff
@@ -0,0 +1,6 @@
+class Admin::WelcomeController < Admin::ApplicationController
+  def index
+    @roasts = Roast.all
```

Comment:
Not sure these will be needed but understand you have them just for debug purposes...